functions/src/index.ts

```diff
@@ -277,3 +277,18 @@ app.post(
       });
   }
 );
+
+/** 
+ * Returns all the registered API keys and their properties. 
+ * 
+ * Requires Bearer authorization token with a valid Firebase ID token.
+ *
+ */
+app.get("/auth/allApiKeys", isFirebaseAuthorized, (res: Response) => {
```

Comment:
Having this endpoint allows us to do something like shown below, which would be nice to have for the UI

<img width="1318" alt="image" src="https://user-images.githubusercontent.com/55333380/204397656-2578cc37-6c0c-4c4c-9bae-ca57ebc69e96.png">